sympy/physics/tests/test_physics_matrices.py

```diff
@@ -69,9 +71,12 @@ def test_Dirac():
         mgamma(0, True)*mgamma(1, True)*mgamma(2, True)*mgamma(3, True)*I
 
 def test_mdft():
-    assert mdft(1) == Matrix([[1]])
-    assert mdft(2) == 1/sqrt(2)*Matrix([[1,1],[1,-1]])
-    assert mdft(4) == Matrix([[S.Half,  S.Half,  S.Half, S.Half],
-                              [S.Half, -I/2, Rational(-1,2),  I/2],
-                              [S.Half, Rational(-1,2),  S.Half, Rational(-1,2)],
-                              [S.Half,  I/2, Rational(-1,2), -I/2]])
+    with raises(SymPyDeprecationWarning):
+        assert mdft(1) == Matrix([[1]])
```

Comment:
When running under pytest warnings are not raised so this test would fail. To test that a deprecation warning is raised we have the `warns_deprecated_sympy` context manager.

Reply:
Seems like the test passes. 

Could be good to update https://github.com/sympy/sympy/wiki/Deprecating-policy then, which is where I read. (Although I noted that it wasn't really used in the code...)

Reply:
If you run it under pytest it fails:
```
$ pytest sympy/physics/tests/test_physics_matrices.py
```

Reply:
I updated the wiki. Thanks for pointing that out because this comes up regularly and I never understood why (I already removed all instances of using raises like this from the codebase ages ago but they keep coming back!).

Reply:
OK! I was surprised that I didn't find any instances of the Wiki example in the code, but that explains it. I'll sort this one out.